src/components/molecules/Visualizer/Engine/Cesium/hooks.ts

```diff
@@ -172,12 +172,18 @@ export default ({
   }, []);
 
   // call onCameraChange event after moving camera
+  const emittedCamera = useRef<Camera[]>([]);
   const updateCamera = useCallback(() => {
     const viewer = cesium?.current?.cesiumElement;
     if (!viewer || viewer.isDestroyed() || !onCameraChange) return;
 
     const c = getCamera(viewer);
     if (c && !isEqual(c, camera)) {
+      emittedCamera.current?.push(c);
+      // an experience value base on how many data version global state can be lag behind
```

Comment:
Sorry, can you try to re-write this? I don't understand what you mean.

Reply:
Sorry for my poor english. I think we need a comment here to describe where the `6` comes from. Updating now.